libopenage/gui/guisys/public/gui_renderer.h

```diff
@@ -5,6 +5,10 @@
 #include <memory>
 
 #ifndef __APPLE__
+#ifdef _MSC_VER
```

Comment:
The `libopenage/gui/guisys` deals only with Qt and SDL. It doesn't include anything else from openage or its dependencies like epoxy. The `openage/libopenage/gui/integration` (and `openage/libopenage/gui/integration/private/gui_make_standalone_subtexture.h` in it) is for connecting together Qt and openage, hence it uses epoxy that's used by openage textures.

This #ifdef boilerplate can be probably fixed by including some GL Qt header in `gui_renderer.cpp` file before the `#include "gui_renderer.h"`. Qt basically does the same thing as epoxy.